site.yml

```diff
@@ -14,6 +14,18 @@
   - { role: base }
   - { role: serf }
   - { role: dev }
+  - { role: vagrant }
+
+- hosts: build
```

Comment:
hmm, `dev` role was needed to pre-install binaries in the packer image but with the `build` host group not using `dev` role those binaries will still need to be pulled in our packer boxes.

I think if we take care of the `ceph-install` role as described above, then the issue that will remain is to avoid installing VBox, vagrant, packer etc that `dev` role brings in but breaks box images. I had added those binaries as I was hoping that I will be able to use `devtest` role for lab servers as well, but looks like that was a bad idea :) .. I am thinking may be for now we should just get rid of vagrant, vbox and packer tasks from dev role altogether then our boxes will be built with the bare minimal set of binaries that we need. WDYT? I will share a gist about what I have in mind as well in a bit.